catalyst/dl/callbacks/optimizer.py

```diff
@@ -23,7 +23,8 @@ def __init__(
         accumulation_steps: int = 1,
         optimizer_key: str = None,
         loss_key: str = "loss",
-        decouple_weight_decay: bool = True
+        decouple_weight_decay: bool = True,
+        return_model_grads: bool = False
```

Comment:
don't you mind to rename this feature to `save_model_grads`? or `dump_model_grads`?
I just think about naming clearance in this case and from my perspective `return` - is something different, @ogvalt @TezRomacH what do you think about it?

Reply:
My motivation to call it ```return_model_grads``` was that Optimizer would return grads after each batch each time in ```RunnerState```. But, since we rather store grads in ```RunnerState``` it would be better to name is ```save_model_grads``` or ```store_model_grads```. ```dump_model_grads``` is more about storing them somewhere in file system, but separate callback will deal with this.

Reply:
deal, `save_model_grads` sounds great for me

Reply:
hi, 
looking forward for renaming and ready for merge :)

Reply:
Done, waiting for checks completed